core/commonMain/src/kotlinx/serialization/builtins/BuiltinSerializers.kt

```diff
@@ -217,3 +218,9 @@ public fun UByte.Companion.serializer(): KSerializer<UByte> = UByteSerializer
 @ExperimentalSerializationApi
 @ExperimentalUnsignedTypes
 public fun UShort.Companion.serializer(): KSerializer<UShort> = UShortSerializer
+
+/**
+ * Returns serializer for [Duration].
+ * By default, it is serialized as a string that represents a duration in ISO-8601 format.
+ */
+public fun Duration.Companion.serializer(): KSerializer<Duration> = DurationSerializer
```

Comment:
Shouldn't it be experimental?

Reply:
@qwwdfsad

Reply:
It cannot be experimental at the moment it has built-in plugin support -- users won't get experimenta warnings on `@Serializable class D(val d: Duration)`, rendering our own experimental warning here useless